core/src/main/java/org/epcc/ps/core/algorithm/CoreAlgorithm.java

```diff
@@ -0,0 +1,26 @@
+package org.epcc.ps.core.algorithm;  
```

Comment:
check for 'rename git commit message' and 'git force push' to make the commits message correct